analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SonarAnalysisContextTest.cs

```diff
@@ -224,6 +304,9 @@ public void ProjectConfiguration_WhenInvalidXml_ThrowException()
                 .WithMessage("File SonarProjectConfig.xml has been added as an AdditionalFile but could not be read and parsed.");
         }
 
+        private static IEnumerable<MetadataReference> ConcatReferences(IEnumerable<MetadataReference> additionalReferences) =>
```

Comment:
~~The `Concat` was puzzling on the usage side, as it takes only one argument.~~
_Edit:_ See next message in this thread.
```suggestion
        private static IEnumerable<MetadataReference> BuildReferences(IEnumerable<MetadataReference> additionalReferences) =>
```
or
```suggestion
        private static IEnumerable<MetadataReference> CreateReferences(IEnumerable<MetadataReference> additionalReferences) =>
```
or
```suggestion
        private static IEnumerable<MetadataReference> BuildAllReferences(IEnumerable<MetadataReference> additionalReferences) =>
```
or
```suggestion
        private static IEnumerable<MetadataReference> CreateAllReferences(IEnumerable<MetadataReference> additionalReferences) =>
```

Reply:
Let's remove the method from this class and do the magic inside the suggested `TestSetup` constructor. That will make `TestSetup.AdditionalReferences` usable right away.